adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/GenotypeRDD.scala

```diff
@@ -159,17 +162,39 @@ case class GenotypeRDD(rdd: RDD[Genotype],
   /**
    * Explicitly saves to VCF.
    *
+   * @param args Arguments defining where to save the file.
+   * @param stringency The validation stringency to use when writing the VCF.
+   *   Defaults to LENIENT.
+   */
+  def saveAsVcf(args: ADAMSaveAnyArgs,
```

Comment:
I believe it is a bit more complex than that, and would be best to handle after #1324 merges (since you lose partitioning on "externally sorted" data).